rotkehlchen/db/filtering.py

```diff
@@ -312,6 +331,12 @@ def make(
         else:
             if addresses is not None:
                 filter_query.join_clause = DBETHTransactionAddressFilter(and_op=False, addresses=addresses)  # noqa: E501
+            elif filter_by_events:
```

Comment:
Hmm so what happens if we want to filter both by address and either asset and/or protocol?

Reply:
This is a question that we with @yabirgb spent time on and came up with this solution. There can be only one `join_clause` and we decided that it will be too much to alter filter classes structure.

Reply:
You can use the location label field of the history events. For eth transacctions is the user address